net/tcp/Kconfig

```diff
@@ -50,11 +50,11 @@ config NET_TCPURGDATA
 		compiled in. Urgent data (out-of-band data) is a rarely used TCP feature
 		that is very seldom would be required.
 
-config NET_TCP_CONNS
-	int "Number of TCP/IP connections"
-	default 8
+config NET_TCP_CONNS_PER_BLOCK
+	int "Number of TCP/IP connections per block(one for each open)"
+	default 4
```

Comment:
let's default to 1 when DEFAULT_SMALL is true

Reply:
Done